### ISSUE TITLE
Copying template files... Command npm failed with exit code 1

### ISSUE THREAD

**Issue by JimLynchCodes** [NativeScript/NativeScript#9261]
My project all of the sudden no longer compiles...

When running either "ns run ios --force" or "ns run android --force" I get this error:

```
Copying template files...
Command npm failed with exit code 1
```

ns info

✔ Getting NativeScript components versions information...
⚠ Update available for component nativescript. Your current version is 7.1.2 and the latest available version is 7.2.0.
⚠ Update available for component @nativescript/core. Your current version is 7.1.4 and the latest available version is 7.3.0.
⚠ Update available for component tns-ios. Your current version is 1.0.3 and the latest available version is 6.5.4.
⚠ Update available for component tns-android. Your current version is 1.0.3 and the latest available version is 6.5.3.

ns doctor

**Comment by mBaldasty** [comment]
Just out of curiosity (and to eliminate the obvious things first): are the listed tns-ios and tns-android the correct once?

**Comment by NathanWalker** [comment]
We're going to close this as the difference in reported runtime version is massive:
```
⚠ Update available for component tns-ios. Your current version is 1.0.3 and the latest available version is 6.5.4.
⚠ Update available for component tns-android. Your current version is 1.0.3 and the latest available version is 6.5.3.
```
